src/hdtSkinnedMesh/hdtSkinnedMeshWorld.cpp

```
@@ -11,19 +11,46 @@
 
 namespace hdt
 {
+	namespace
+	{
+		// Todo: Look into using physical cores here rather than threads. By default, this will use threads
+		int initBulletTbbAndGetThreadCount()
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)